lib/Partitioner/Partitioner.cpp

```diff
@@ -19,6 +19,22 @@
 using namespace glow;
 using llvm::isa;
 
+/// Check if the memory of \p node inputs is calculated already. \p return the
```

Comment:
\p stands for "parameter". No need for \p before the return. Use "\returns". 

Reference and examples here:

http://www.doxygen.nl/manual/docblocks.html